code/game/machinery/lightswitch.dm

```diff
@@ -61,3 +65,23 @@
 		return
 	if(!(stat & (BROKEN|NOPOWER)))
 		power_change()
+
+/obj/machinery/light_switch/proc/shock(mob/user)
+	if(stat & (BROKEN|NOPOWER))		// unpowered, no shock
+		return FALSE
+	var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread
+	s.set_up(5, 1, src)
+	s.start()
+	electrocute_mob(user, get_area(src), src, 0.7, TRUE)
+		return TRUE
+	else
+		return FALSE
```

Comment:
Remove these 3 lines, that's what alex said.

Reply:
Commited. Also fixed compiling issues.
Thanks